### DISCUSSION TITLE
I can't change language in "admin/localization"

### DISCUSSION THREAD

**Discussion by cnah66** [snipe/snipe-it#10472] [Q&A] [answered]
I change language to English , and then save, but the web language are still Chinese ,  I have to edit .env file?

![改不了语言](https://user-images.githubusercontent.com/95849938/147562231-d552d64f-f756-472e-878f-513739fb7666.png)

**Comment by King-GO** [comment, marked as answer]
edit your profile >Language > English

**Reply by cnah66** [reply]
Thank you for your help!
